### DISCUSSION TITLE
Two-way syncing?

### DISCUSSION THREAD

**Discussion by Torfstecher** [phw198/OutlookGoogleCalendarSync#1966] [Q&A]
Yesterdy I have installed OGCS 2.11.0.0
As an ignoramus may I kindly ask the community to discuss the following experience with OGCS. 
Syncing O - G works fine, syncing G - O fails. I am exspecting an entry in G will appear in O. In addition a deleted an entry in O must be deleted manually in G.

**Comment by phw198** [comment]
What's the failure message?
Have you disabled deletions?

**Reply by Torfstecher** [reply]
Here is the message: 
1 Google items would have been deleted, but you have deletions disabled.
Unfortunately I have no clue where and how I can deletions enable.

![google Meldung](https://github.com/user-attachments/assets/e69dddee-3a96-41aa-aa27-7653ae952a86)

**Reply by Torfstecher** [reply]
I must confess that I am really frustrated. Each time when I want to figure out the outcome of a different setting of OGCS the list of my Google Calender increases with identical appointments. I have deinstalled and reinstalled the GC but when opened nothing has changed.  
Where and how can I delete all the entries?

**Reply by phw198** [reply]
Settings > Sync Options > How > Uncheck "Disable Deletions"

**Reply by Torfstecher** [reply]
Thank you for your reply. 
Later have found that my question was really unnecessary. I have done it several times before. 
In the mean time I have uninstalled OGCS and I am using now the Google Calender Abo which satifies my needs and is working fine.
Bye!
